src/index.js

```diff
@@ -15,21 +21,23 @@ Promise.all([roomsData, usersData, bookingsData])
     roomsData = data[0];
     usersData = data[1];
     bookingsData = data[2];
-  })
-  .then(() => {
-    checkLogin(roomsData, usersData, bookingsData);
-  })
-
-function checkLogin(roomsData, usersData, bookingsData) {
-  console.log(roomsData);
-  console.log(usersData);
-  console.log(bookingsData);
+  }).then(() => {
+    $('.explore-button').click(function() {
+      rooms = new Rooms(roomsData, bookingsData)
+      checkLoginInfo(roomsData, usersData, bookingsData);
+    })
+})
+
+function checkLoginInfo(roomsData, usersData, bookingsData) {
+  let userId = parseInt($('#username-input').val().slice(8));
+  let passwordCheck = $('#password-input').val() === 'overlook2019';
+  let userIdCheck = Number.isInteger(userId);
```

Comment:
Just realized it would make a lot more sense and work 100% of the time to check through the usersData and find a matching id rather than this... Will change later.